config/templates/stack-template.json

```diff
@@ -742,7 +754,13 @@
         "SecurityGroups" : [
           { "Ref" : "SecurityGroupElbAPIServer" }
         ]
-      }
+      },
+      "DependsOn" : [
+        {{range $etcdIndex, $etcdInstance := .EtcdInstances}}
+        {{if gt $etcdIndex 0}},{{end}}
+        "InstanceEtcd{{$etcdIndex}}"
+        {{end}}
+      ]
```

Comment:
I found that when running within a small private VPC the controller ELB will provision faster than the etcd nodes and more often than not will be provisioned with the same IP that was calculated for one of the etcd nodes resulting in a failed deployment.